modules/ROOT/pages/import-api-specification-exchange.adoc

```diff
@@ -28,7 +28,7 @@ To list all available APIs, type a `*` character in the search bar and press Ent
 . To select the version of the API to import, use the *Selected Modules* table.
 . Click *Finish*.
 
-Studio scaffolds your API specification. If you choose not to create flows out of your API specification, unselect *Scaffold Flows From These API Specifications*.
+Studio scaffolds your API spec. If you choose not to create flows out of your API specification, unselect *Scaffold Flows From These API Specifications*.
```

Comment:
```suggestion
By default, Studio scaffolds your API spec. To prevent Studio from creating flows out of your API specification, unselect *Scaffold Flows From These API Specifications*.
```

Reply:
OK! Thanks!